templates/DevTools.ss

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset=utf-8/>
+    <meta name="viewport" content="user-scalable=no, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, minimal-ui">
+    <title>GraphQL IDE | Silverstripe CMS</title>
+    <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/graphql-playground-react/build/static/css/index.css" />
+    <link rel="shortcut icon" href="//cdn.jsdelivr.net/npm/graphql-playground-react/build/favicon.png" />
+    <script src="//cdn.jsdelivr.net/npm/graphql-playground-react/build/static/js/middleware.js"></script>
+</head>
+
+<body>
+<div id="root">
+    <style>
+        body {
+            background-color: rgb(23, 42, 58);
+            font-family: Open Sans, sans-serif;
+            height: 90vh;
+        }
+
+        #root {
+            height: 100%;
+            width: 100%;
+            display: flex;
+            align-items: center;
+            justify-content: center;
+        }
+
+        .loading {
+            font-size: 32px;
+            font-weight: 200;
+            color: rgba(255, 255, 255, .6);
+            margin-left: 20px;
+        }
+
+        img {
+            width: 78px;
+            height: 78px;
+        }
+
+        .title {
+            font-weight: 400;
+        }
+    </style>
+    <img src='//cdn.jsdelivr.net/npm/graphql-playground-react/build/logo.png' alt=''>
+    <div class="loading"> Loading
+        <span class="title">GraphQL Playground</span>
+    </div>
+</div>
+<script>
+    var SECURITY_ID = '$SecurityID';
```

Comment:
Shouldn't inline JS as it's not good for CSP.  Some browsers may be configured to not execute inline scripts at all, or at very least they'll throw a bunch of warnings in the console.  Put this into a javascript include instead.

May as well do the same for the CSS at the same.

Reply:
Sorry, but we do this all over the show in the CMS. Browsers aren't configured to deny inline scripts, it's the HTTP response headers. And those would be set on a controller level (e.g. `Page_Controller`). If you set this globally in `.htaccess` then you'd also break the CMS.

Making this a separate route which dynamically returns JSON just seems to open more potential for security issues than it solves here. And it increases load times on the view. I'd say we solve this at the same time as we address it in the CMS, potentially via a JWT-based login token shared between CMS and other routes

Reply:
`/dev/graphql/ide` isn't in the CMS though?  It's being served as something standalone in the frontend, similar to a Page?

Seems like we should be going in a security focused direction here and give ourselves the option of adding CSP headers via the controller that serves this, either now or later?

Reply:
Keep in mind we're in the context of `DevelopmentAdmin`, here. If it's secure enough for CMS, I'd say it's secure enough for DevelopmentAdmin, wouldn't you agree?

Reply:
Yeah, let's not do this, it's overkill.